content/v2.0/get-started.md

```diff
@@ -194,6 +195,100 @@ docker exec -it influxdb /bin/bash
 {{% /tab-content %}}
 <!--------------------------------- END Docker -------------------------------->
 
+<!-------------------------------- BEGIN kubernetes---------------------------->
```

Comment:
I like the step-by-step model you’ve used on this tab. Eventually, let's use same model across tabs for logical and visual consistency (as is, feels off). For now, we cld:

- add a 2nd level heading to match “Set up InfluxDB”, perhaps "Download and install minikube"
- add overview bullets at top to identify what to do to get started, before getting into steps (eventually, this applies across tabs):
   - Download and install minikube
   - Set up InfluxDB 


Notice the macOS tab and Linux tab have “Start InfluxDB” does that step need to happen for Docker and Kubernetes?